### ISSUE TITLE
Zoekbalk type-ahead (suggesties) verwerken in toolkit 

### ISSUE THREAD

**Issue by MichielDHVD** [dso-toolkit/dso-toolkit#366]
Vanuit de viewer is het verzoek om de Type-ahead zo als die nu is gebouwd in. 

https://acc.omgevingswet.overheid.nl/checken

@tfrijsewijk Is deze typeahead duidelijk genoeg om te verwerken in de toolkit of heb je nog extra specs nodig? 

@DjangoBlok

**Comment by bramhoosemans** [comment]
Dit component heeft nog verdere afstemming tussen designers nodig.

**Comment by tfrijsewijk** [comment]
Dit lijkt een vervolg op #6, wellicht dat daar nog interessante overwegingen in staan

**Comment by bramhoosemans** [comment]
De wens is om het zoekbalk component uit te breiden met 'suggesties' en voorbeeld van toegankelijke markup (zover dit niet al gedaan is) op te nemen in de toolkit. Zie specificaties in bijgevoegde export. Een werkende implementatie van de huidige situatie is te vinden op https://acc.omgevingswet.overheid.nl/checken.

**Specificaties:**
[Zoekbalk suggesties.zip](https://github.com/dso-toolkit/dso-toolkit/files/3574282/Zoekbalk.suggesties.zip)

![Zoekbalk suggesties](https://user-images.githubusercontent.com/598735/64251088-f69a1000-cf17-11e9-8c0a-266cbf244fc8.png)

**Comment by tfrijsewijk** [comment]
![image](https://user-images.githubusercontent.com/9696261/66955958-bc1daa00-f063-11e9-9f34-b5f1333f0246.png)

Ik probeer de typeahead in het DOM te krijgen met een zoekopdracht, alleen faalt de backend waardoor ik nu niet verder kan. Wie kan hier wat aan doen?

**Comment by MichielDHVD** [comment]
Ik weet niet goed wat je probeert te doen. Als ik naar de acc omgeving ga kan ik wel een zoekopdracht uitvoeren.

**Comment by MichielDHVD** [comment]
Een korte samenvatting van recente ontwikkelingen. 

Het probleem met de typeahead is dat het niet Frame work agnostisch ontwikkeld kan worden. De toolkit is zo ontworpen om gebruikt te kunnen worden ongeacht de programmeer taal die developers gebruiken. 

Nu is de typeahead die gebouwd is door PR12 is een uitstekend voorbeeld van een werkend typeahead met extra aandacht voor screenreaders. Alleen is de implementatie van de typeahead sterk afhankelijk van react van pr12. 

Wat wel geleverd kan worden is een Documentatie pagina die precies het gedrag en gebruik van de typeahead omschrijft. Zodat andere teams een leidraad hebben om toch het zelfde component met dezelfde look en feel te kunnen bouwen. 

Hier aan gekoppeld zal een sketch bestand zijn. Zodat duidelijk de afmetingen, kleurgebruik gecommuniceerd kunnen worden voor de implementatie.

**Comment by MichielDHVD** [comment]
Verplaatst deze naar volgende milestone. Moet nog meer besproken worden vooral het verschil tussen. Zoeksuggestie en filteren oplijst documentatie

**Comment by MichielDHVD** [comment]
Deze pull request staat al een tijdje open. De inhoud is volledig afgestemd en als er geen technische bezwaren zijn kan deze gemerged worden.

**Comment by tfrijsewijk** [comment]
Dat dit PR al een tijdje staat komt hierdoor:

![image](https://user-images.githubusercontent.com/9696261/74843217-e641ea00-532b-11ea-9e16-9c6cfb729dd1.png)

Zoals aangegeven over Slack is er een probleem met het renderen van het font in de SVG. Je maakt hier een PNG van. Ik merge zodra de PNG is ingecheckt
